File: April 28, 2026.py
def get_number_words(n):

    if n == 0:
        return "zero"
    
    unique_names = [
        "one", "two", "three", "four", "five",
        "six", "seven", "eight", "nine", "ten",
        "eleven", "twelve", "thirteen", "fourteen", "fifteen",
        "sixteen", "seventeen", "eighteen", "nineteen"
    ]

    units = [
        "", "one", "two", "three", "four",
        "five", "six", "seven", "eight", "nine"
    ]

    tens = [
        "twenty", "thirty", "forty",
        "fifty", "sixty", "seventy", "eighty", "ninety"
    ]

    if n > 0 and n < 20:
        return unique_names[n - 1]

    number_str = ""

    if n > 20 and n < 100:
        decimal = n // 10
        unit = n % 10

        if unit > 0:
            number_str = tens[decimal - 2] + "-" + units[unit]
        else:
            number_str = tens[decimal - 2]

    return number_str

File: April 28, 2026.js
function getNumberWords(n) {

  if (n == 0) return "zero";
  
  const uniqueNames = [
  "one", "two", "three", "four", "five",
  "six", "seven", "eight", "nine", "ten",
  "eleven", "twelve", "thirteen", "fourteen", "fifteen",
  "sixteen", "seventeen", "eighteen", "nineteen"
  ];

  const units = [
  "", "one", "two", "three", "four",
  "five", "six", "seven", "eight", "nine"
  ];

  const tens = [
  "twenty", "thirty", "forty",
  "fifty", "sixty", "seventy", "eighty", "ninety"
  ];

  if (n > 0 && n < 20) {
    return uniqueNames[n-1];
  }

  let numberStr = "";

  if (n > 20 && n < 100) {
    let decimal = Math.floor(n / 10);
    let unit = n % 10;
    if (unit > 0) {
      numberStr = tens[decimal-2] + "-" + units[unit]
    } else {
      numberStr = tens[decimal-2];
    }
    
  }

  return numberStr;
}